lib/rspec-puppet/setup.rb

```diff
@@ -55,7 +55,11 @@ def self.get_module_name
     end
 
     def self.get_module_name_from_file(file)
-      p = Puppet::Parser::Lexer.new
+      if Puppet.version.to_f >= 4.0
+        p = Puppet::Pops::Parser::Lexer2.new
```

Comment:
@hlindberg is there any less worse way to do that?
is that interface even supposed to be used / "public" / "documented" ?